src/orc/datamodel/models.py

```diff
@@ -33,3 +35,34 @@ class Meta:
 
     def __str__(self):
         return self.identificatie
+
+
+class NietNatuurlijkPersoon(models.Model):
+    uuid = models.UUIDField(default=_uuid.uuid4)
+
+    rsin = models.CharField(
```

Comment:
er zit in `zds-schema` een `RSINField`, alleen heeft die een lengte van 9 karakters. Wat is het verschil?

Reply:
RGBZ en RSGB nagekeken - enige wat erop lijkt is 'identificatie' in RSGB, en in RGBZ heet het effectief RSIN. Kan geen motivering vinden voor de 17 karakters.